lib/pdn/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2010 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config PDN_LIBRARY
+	bool "PDN library interface for nRF9160"
+	depends on BSD_LIBRARY
```

Comment:
Looking at the implementation, there should also be a dependency to `AT_CMD`/`AT_NOTIF`.

Reply:
true, will be fixed. thanks